README.md

```diff
@@ -1,16 +1,17 @@
 # imagemagick-build
 
-This repo is building ImageMagick v7 packages for Rocky Linux 8, Centos7, Ubuntu-18.04 & Ubuntu20.04 that are
+This repo is building ImageMagick v7 packages for [RockyLinux8](https://github.com/Alfresco/imagemagick-build/tree/main/.github/actions/rockylinux-build), [Centos7](https://github.com/Alfresco/imagemagick-build/tree/main/.github/actions/centos-build), [Ubuntu-18.04](https://github.com/Alfresco/imagemagick-build/tree/main/.github/actions/Ubuntu18.04-build), [Ubuntu20.04](https://github.com/Alfresco/imagemagick-build/tree/main/.github/actions/Ubuntu20.04-build) & [Ubuntu22.04](https://github.com/Alfresco/imagemagick-build/tree/main/.github/actions/Ubuntu22.04-build) that are
 not available elsewhere and that are required by
 [Alfresco](https://docs.alfresco.com/content-services/latest/support/) to work properly.
 
-Note:- ImageMagick v7 for Centos7 & Rockylinux 8 are the generic packages which can be used for Redhat7 & Redhat8 respectively.
+Notes:- 
+
+* ImageMagick v7 for [Centos7](https://github.com/Alfresco/imagemagick-build/tree/main/.github/actions/centos-build) & [Rockylinux8](https://github.com/Alfresco/imagemagick-build/tree/main/.github/actions/rockylinux-build) are the generic packages which can be used for Redhat7 & Redhat8 respectively.
 
 Packages are published under [Releases](https://github.com/Alfresco/imagemagick-build/releases) section and on our [Nexus](https://nexus.alfresco.com/nexus/service/local/repositories/thirdparty/content/org/imagemagick/imagemagick-distribution/) instance.
 
 ## Release a new version
 
-* Bump ImageMagick env var version in [imagemagick-version file](https://github.com/Alfresco/imagemagick-build/blob/main/.github/actions/imagemagick-version) via PR
-* Create a new tag `git tag -s vN.N.N -m vN.N.N` following semantic versioning
-* Push it `git push --tags origin vN.N.N`
-* Draft a [new release](https://github.com/Alfresco/imagemagick-build/releases/new) using the previous tag as source and use the `Generate release notes` function to generate the changelog.
+* Bump ImageMagick env var version in [Dockerfile](https://github.com/Alfresco/imagemagick-build/blob/main/.github/actions/rockylinux-build/Dockerfile#L3)
+* Tag the commit via `git tag -s v7.1.0-16 -m v7.1.0-16`
+* Push it `git push --tags origin v7.1.0-16`
```

Comment:
you did the opposite during the merge, overwriting changes I did in my last PR, restore this